presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMaterializedViewLogicalPlanner.java

```diff
@@ -1232,10 +1229,7 @@ public void testBaseToViewConversionCountOptimizationWithFreshView()
                                                                     values("orderkey", "ds"))))),
                                     anyTree(
                                             //expect no scan to happen over the base table since materialized view is completely fresh
-                                            constrainedTableScan(
-                                                    view,
-                                                    ImmutableMap.of("ds", multipleValues(createVarcharType(10), utf8Slices("2021-07-11", "2021-07-12"))),
-                                                    ImmutableMap.of())))));
+                                            constrainedTableScan(view, ImmutableMap.of(), ImmutableMap.of())))));
```

Comment:
What happened to the original map values?

Reply:
I removed them per @jainxrohit 's advice, because we aren't depending on explicitly scanning these partitions from the MV. For example, we could select all fresh partitions from the MV, which would scan the same rows but fail this test due to the domains not being explicitly selected. The main thing we are concerned about is the domain being scanned from the base table, which is still in the plan validation.